GenericDiffractometer.py

```diff
@@ -283,6 +290,36 @@ def init(self):
                 self.use_sc = True
 
 
+        # centring motors
+        self.centring_motors_list = ["phiz", "phiy","sampx","sampy"]
+	try:
+           self.use_sample_centring = self.getProperty("sample_centring")
```

Comment:
From my point of view this should not be in GenericDiffractometer, because we use different centering algorithm (CentringMath) which I import in EMBLMiniDiff